sources/packages/backend/apps/api/src/route-controllers/designation-agreement/_tests_/e2e/designation-agreement.institutions.controller.submitDesignationAgreement.e2e-spec.ts

```diff
@@ -101,6 +117,58 @@ describe("DesignationAgreementInstitutionsController(e2e)-submitDesignationAgree
     );
   });
 
+  it("Should return an unprocessable entity when a new designation is requested and there is already a pending designation agreement.", async () => {
+    // Arrange
+    const payload = {
+      dynamicData: {
+        eligibilityOfficers: [],
+        enrolmentOfficers: [],
+        scheduleA: false,
+        scheduleB: false,
+        scheduleD: false,
+        legalAuthorityName: "SIMS COLLC",
+        legalAuthorityEmailAddress: "test@gov.bc.ca",
+        agreementAccepted: false,
+      },
+      locations: [
+        {
+          locationId: collegeCLocation.id,
+          requestForDesignation: true,
+        },
+      ],
+    };
+    const formService = await getProviderInstanceForModule(
+      testingModule,
+      AppInstitutionsModule,
+      FormService,
+    );
+    formService.dryRunSubmission = jest
+      .fn()
+      .mockResolvedValue({ valid: true, data: { data: payload } });
+    const institutionUserToken = await getInstitutionToken(
+      InstitutionTokenTypes.CollegeCAdminLegalSigningUser,
+    );
+    const endpoint = "/institutions/designation-agreement";
+
+    // Act/Assert
+    let designationAgreementId: number;
```

Comment:
`designationAgreementId` is not used anywhere.

Reply:
removed